packages/api/models/nomination.js

```diff
@@ -35,10 +34,10 @@ module.exports = (sequelize, DataTypes) => {
       representativeFirstName: DataTypes.STRING,
       representativeLastName: DataTypes.STRING,
       representativeEmailAddress: DataTypes.STRING,
-      relationship: DataTypes.STRING,
-      patientFirstName: DataTypes.STRING,
-      patientLastName: DataTypes.STRING,
-      age: DataTypes.INTEGER,
+      representativeEmailPhoneNumber: DataTypes.INTEGER,
```

Comment:
the migration is failing because this column is not in the migrations!

Reply:
![image](https://user-images.githubusercontent.com/1637481/96067692-f03acf00-0e5f-11eb-87cf-911d4e9f164e.png)

Reply:
ahhhhhhh

Reply:
fixed